src/rotator_library/anthropic_compat/streaming_fast.py

```
@@ -0,0 +1,460 @@
+OpenAI SSE (Server-Sent Events) format to Anthropic's streaming format.
+
+Performance optimizations:
+- Uses orjson for fast JSON parsing (3-5x faster than stdlib json)
+- Reuses parsed objects where possible
+- Minimizes allocations in hot paths
+- Pre-built templates for common events
+"""
+
+import logging
+import uuid
+from typing import AsyncGenerator, Callable, Optional, Awaitable, Any, TYPE_CHECKING
```

Ruff (0.15.4)

[warning] Import from `collections.abc` instead: `AsyncGenerator`, `Callable`, `Awaitable`

Import from `collections.abc`

(UP035)